pvnet/models/multimodal/multimodal.py

```diff
@@ -148,13 +151,15 @@ def __init__(
         self.min_sat_delay_minutes = min_sat_delay_minutes
         self.adapt_batches = adapt_batches
 
-        if location_id_mapping is None and add_image_embedding_channel:
-            raise ValueError(
-                "If `add_image_embedding_channel` is set to True, `location_id_mapping` must be "
-                "provided."
-            )
+        if location_id_mapping is None:
+            logger.warning("location_id_mapping` is not provided, we'll set a default")
+
```

Comment:
I think this should say what it's defaulting to
```suggestion
            logger.warning("location_id_mapping` is not provided, defaulting to outdated GSP mapping (1 to 318)")

```